lib/ensureAfterHookIsRegistered.js

```diff
@@ -289,7 +294,20 @@ function ensureAfterBlockIsRegistered(topLevelFixes) {
         fixesByFileName[fileName].map(fix => ({ fix }))
       );
       if (fixResult.fixed) {
-        fixedSourceTextByFileName[fileName] = fixResult.output;
+        let fixedSource = fixResult.output;
+        if (prettier) {
+          const prettierConfig = prettier.resolveConfig.sync(fileName);
+          if (prettierConfig) {
```

Comment:
What if I'm using the default prettier without a config file? Wouldn't it be better also format if a prettier can be resolved?

Reply:
I was mostly trying to be a bit conservative :laughing:

Reply:
No needed for that 😅

Reply:
It seems to work, but I'll need to think of how to get the test suite to run. It breaks 10 other tests because prettier is available.

Reply:
Opt out environment variable?

Reply:
I checked, and there's no standard `PRETTIER*` environment variable for it, so it would only really be for the test suite.

Reply:
Fixed in 6b9b884, does that seem to work?